extras/reimbursement_tutorial.md

```diff
@@ -0,0 +1,130 @@
+# Filling Out A Reimbursement 101
+
+This is a tutorial on how to fill out a reimbursement. May it be for travel or for hosting a candidate interview dinner.
+For this tutorial we will assume that we are reimbursing a postdoc candidate interview dinner.
+
+## Fill Out Agreement Form
+
+### Graduate Student
+
+First step is to fill out an [agreement form](https://www.finance.upenn.edu/sites/default/files/Penn-Student-Agreement.pdf). 
+This form is essential for getting any reimbursements accept.
+**Any submitted report that misses this form will bounce back**.
+Filling out this form is pretty straight forward.
+For the scope of work section fill out topic keywords that describe of your thesis project (PhD Candidates) or current research (Pre-Candidacy Students).
+In the Term section fill out your **projected graduation year plus one** (Just to be safe given the joys of gradudate school).
+Lastly, sign your name by the Student/Trainee section and give your last 4 digits of your social security number.
+Now you are done with this section and can move on to [filling out the reimbursement report](#filling-out-reimbursement-report).
+
+### PostDocs and Staff 
+
+The form and link will be filled out here
+ 
+## Filling Out Reimbursement Report
+
+### Report Header
+This section will show you how to fill out a reimbursement report.
+Open [concur](https://medley.isc-seo.upenn.edu/authentication/profile/concur?app=concurprod) and click on the expense tab.
+Your screen should look like the screenshot below minus the submitted report.
+
+![opening screen](tutorial_screenshots/new_report_screen.png)
+
+Click on the create new report to get started on filling the reimbursement form out.
+Your screen should look like below.
+
+![report header](tutorial_screenshots/blank_report_header.png)
+
+Now the fields highlighted in red are what you need to fill out. 
+**Type of Travel/Reimbursement**, **Report Name** and **Report Date** are straight forward to fill out.
+The policy field should be automatically filled, so don't worry about that.
+Now an important field to fill out is the **Business Justification**.
+In this field you need to provide justification for why you should be reimbursed.
+Rule of thumb: more detail you prove the better.
+For example, if this is a travel reimbursement for a conference, you *should* provide the following information:
+
+1. Name of Conference
+2. Conference Location
```

Comment:
Do you also have to put in the dates? We do.

Reply:
Usually the date is in the report header, but I added it to the instructions just to be on the safe side.